crypto/objects/o_names.c

```diff
@@ -111,8 +110,8 @@ int OBJ_NAME_new_index(unsigned long (*hash_func) (const char *),
             ret = 0;
             goto out;
         }
-        name_funcs->hash_func = OPENSSL_LH_strhash;
```

Comment:
Yes as both `OPENSSL_LH_strhash` and as `lh_strhash` :(

Given how close to release we are, I'd lean towards not deprecating it this time around.

Reply:
OTOH one could argue that if we don't add the deprecated definition thingy before 1.1.1 release, we might be making users a disservice if we plan to get rid of it as soon as possible (1.2.0?)

Anyway, I'd say this would apply only to 1.1.1, and a separate PR. 

I'll open it and it can be discussed there.